src/user/dto/user.from.db.dto.ts

```diff
@@ -0,0 +1,15 @@
+export class UserFullDto {
+  _id: string;
+  username: string;
+  password: string;
+  isAdmin: boolean;
+  check: string;
+  createdAt: number;
+}
+
+export class UserSafeDto {
```

Comment:
It is not a good approach, the easiest, correct one would be to use Omit on whole UserDto object - https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
and the file should be named user-from-db.dto.ts, use dots to separate important parts of file name like folders or modules they belong to.